packages/daimo-expo-enclave/ios/ExpoEnclaveModule.swift

```diff
@@ -34,6 +34,10 @@ public class ExpoEnclaveModule: Module {
       }
     }
 
+    AsyncFunction("forceFallbackUsage") { () in
+      keyManager = FallbackKeyManager()
```

Comment:
is it possible to put this into a constructor or `init` so that we initialize `keyManager` once?

having this imperative function feels bad, but OK if it's the only way

Reply:
Yeah, I also wanted to do that but afaict we have no control over `ExportedModule` life cycles and the constructor/`init` is called before we can inject custom arguments in runtime.